sed/config/flash_example_config.yaml

```diff
@@ -18,9 +18,9 @@ core:
   # provided, the loader will try to find the data based on year beamtimeID etc
   paths:
     # location of the raw data.
-    data_raw_dir: "tests/data/loader/flash/"
+    raw: "tests/data/loader/flash/"
     # location of the intermediate parquet files.
-    data_parquet_dir: "tests/data/loader/flash/parquet"
+    processed: "tests/data/loader/flash/parquet"
```

Comment:
Misses update of Tutorial

Reply:
Thanks. Also needs updating with the main branch

Reply:
done that.